lib/counter.js

```diff
@@ -25,6 +25,17 @@ function removeComments(str) {
     return str;
 }
 
+/**
+ * Empty the string literals. Do lazy match. Avoids removing
+ * everything after '//' in a string literal.
+ */
+function emptyStringLiterals(str) {
+    str = str || "";
+    str = str.replace(/\"[^\n\r]*?\"/g, "''");
```

Comment:
This is still a useful PR. The regex just needs to be updated. I can do this later.